boards/nxp/frdm_mcxa156/frdm_mcxa156_defconfig

```diff
@@ -10,3 +10,4 @@ CONFIG_SERIAL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_GPIO=y
 CONFIG_LPADC_DO_OFFSET_CALIBRATION=y
+CONFIG_TEST_RANDOM_GENERATOR=y
```

Comment:
This should not be enabled for real hardware by default. How come this was added?